controllers/feeds/getFeedsV2.js

```diff
@@ -19,10 +19,11 @@ const {
   modifyReactionsPost,
   modifyFeedIsFollowingTarget
 } = require('../../utils/post');
-const {DomainPage, Locations, User} = require('../../databases/models');
+const {DomainPage, Locations, User, UserFollowUser} = require('../../databases/models');
 const RedisDomainHelper = require('../../services/redis/helper/RedisDomainHelper');
 const {ACTIVITY_THRESHOLD} = require('../../config/constant');
 const UsersFunction = require('../../databases/functions/users');
+const UserFollowUserFunction = require('../../databases/functions/userFollowUser');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to include `UserFollowUser` and `UserFollowUserFunction`. This is necessary for the new feature of blurring posts for certain users. The changes look good.
```diff
- const {DomainPage, Locations, User} = require('../../databases/models');
+ const {DomainPage, Locations, User, UserFollowUser} = require('../../databases/models');
- const UsersFunction = require('../../databases/functions/users');
+ const UsersFunction = require('../../databases/functions/users');
+ const UserFollowUserFunction = require('../../databases/functions/userFollowUser');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->